js/src/components/paid-ads/validateCampaign.js

```diff
@@ -23,6 +23,26 @@ const validateCampaign = ( values ) => {
 		);
 	}
 
+	if (
+		Number.isFinite( values.amount ) &&
+		Number.isFinite( values.budget?.daily_budget )
+	) {
+		const { budget, budgetMin, amount } = values;
+		const { daily_budget: dailyBudget } = budget;
+		const minAmount = Math.round( dailyBudget * budgetMin, 2 );
+
+		if ( amount < minAmount ) {
+			errors.amount = sprintf(
+				/* translators: %s: minimum daily budget */
+				__(
+					'Please make sure daily average cost is at least %s.',
```

Comment:
@dsawardekar Can you check the AC again for the wording to be used please? I quote:
> Please make sure daily average cost is greater than %d." where %d is replaced with the minimum.

Reply:
@joemcgill Can you kindly advise here please?

Reply:
I don't think we need to explain how we arrived at the minimum value, but instead just need to show them what the minimum value is.